### PR TITLE
bugfix lineage generator

### PR DESCRIPTION
**Pull request by bernt-matthias**
manual rank selection did not work

solves first part of https://github.com/TGAC/earlham-galaxytools/issues/95

### CODE REVIEW

File: tools/ete/ete_lineage_generator.xml [hunk 1]
```diff
@@ -12,9 +12,7 @@ python '$__tool_directory__/ete_lineage_generator.py'
 #if $ranks.levels == "full"
     -f
 #elif $ranks.levels == "manual"
-    #for $rank in $ranks.manualranks:
```
Comment (nsoranzo) [inline tools/ete/ete_lineage_generator.xml:15]: A simpler fix for the bug is to only change this line to:
```
    #for $rank in str($ranks.manualranks).split(','):
```
and restore the Python code.

Also, you should add `optional="false"` to the `manualranks` param, there's no point of running the tool with no rank, right?

**Comment by nsoranzo** [comment]
You need to update the test output data to include the headers.

**Comment by bernt-matthias** [comment]
That's true, but my point is that the headers are not shown as headers in Galaxy.

**Comment by nsoranzo** [comment]
I think that's how tabular datasets work in Galaxy, you should see that the dataset has 1 comment line in the dataset info, but the column headers remain 1, 2, 3...

File: tools/ete/ete_lineage_generator.xml (hunk 1 above)
Comment (bernt-matthias) [reply tools/ete/ete_lineage_generator.xml:15]: done

**Comment by bernt-matthias** [comment]
fixed the bugs and extended the tests .. the tool is finished from my side

**Comment by nsoranzo** [comment]
@bernt-matthias The test data are still missing the header line, can you fix them please?
`planemo test --update_test_data` can help you there.

**Comment by bernt-matthias** [comment]
just forgot to push .. sorry

**Comment by nsoranzo** [comment]
Thanks @bernt-matthias !

**Pull request merged**